### PR TITLE
expose settings in information_schema.partitions

### PR DESCRIPTION
**Pull request by chaudum**
+ alter settings of partitioned table

### CODE REVIEW

File: sql/src/test/java/io/crate/analyze/CreateAlterTableStatementAnalyzerTest.java [hunk 1]
```diff
@@ -678,7 +677,6 @@ public void testChangeReadBlock() throws Exception {
     public void testChangeWriteBlock() throws Exception {
         AlterTableAnalyzedStatement analysis =
                 (AlterTableAnalyzedStatement)analyze("alter table users set (\"blocks.write\"=true)");
-        assertThat(analysis.table().ident().name(), is("users"));
```
Comment (mfussenegger) [inline sql/src/test/java/io/crate/analyze/CreateAlterTableStatementAnalyzerTest.java]: why did you remove these assertions?

Comment (chaudum) [inline sql/src/test/java/io/crate/analyze/CreateAlterTableStatementAnalyzerTest.java]: it's not really the aim of these tests IMO

**Comment by mfussenegger** [comment]
ack
